Automattic/newspack-theme
# Comments: improve appearance of comment log in messages

### All Submissions:


### Changes proposed in this Pull Request:

The theme will display messages when a site requires you to log in to comment; right now, those messages are too big and don't really fit. This PR fine tunes them.

Closes #975.

### How to test the changes in this Pull Request:

1. Under Settings > Discussion, turn on comments and limit them to registered, logged in users. 
2. While logged in, add a comment to a post, so you have the option to reply to it. 
3. View on the front-end in an incognito window:

![image](https://user-images.githubusercontent.com/177561/85461077-c67c3600-b558-11ea-8409-6fae9012c746.png)

4. Apply the PR and run `npm run build`
5. Confirm that the 'Log in to Reply' link is now styled like the 'Reply' link (using the header font), and the 'log in to comment' text at the bottom is a bit smaller, and italicized:

![image](https://user-images.githubusercontent.com/177561/85461167-e7448b80-b558-11ea-94d4-a205dcbc3c90.png)

### Other information:
